controller/controller.go

```diff
@@ -271,16 +271,17 @@ func newDriverTasks(conf *config.Config, providerConfigs driver.TerraformProvide
 // configuration by ID to a driver service type. If a service is not
 // explicitly configured, it assumes the service is a logical service name
 // in the default namespace.
-func getService(services *config.ServiceConfigs, id string) driver.Service {
+func getService(services *config.ServiceConfigs, id string, meta map[string]map[string]string) driver.Service {
```

Comment:
Those map of maps data types can really get verbose. I tend to want to make them into local types to help with readability. Definitely not a big deal though.

Reply:
:nod: i do not like either. Thanks for the nudge, I looked around and saw this nested map in a number of places. Time to typify this! https://github.com/hashicorp/consul-terraform-sync/pull/304/commits/1e686da0eb8c13fa8a9b67d03839d16a0db6f3de